spec/routing/users_routing_spec.rb

```diff
@@ -1,31 +1,29 @@
-require "rails_helper"
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.